e2e/node_test.go

```diff
@@ -105,10 +105,23 @@ func testNode() {
 				"--nosuffix",
 				vgName,
 			)
+
 			Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", targetBytes, stderr)
-			val, ok := node.Annotations[topolvm.GetCapacityKeyPrefix()+"ssd"]
+
+			free, err := strconv.Atoi(strings.TrimSpace(string(targetBytes)))
+			Expect(err).ShouldNot(HaveOccurred())
+
+			// spare-gb is set to 1 in the manifests
+			spare := 1 << 30
+			if free < spare {
+				free = 0
+			} else {
+				free -= spare
+			}
+
+			val, ok := node.Annotations[topolvm.CapacityKeyPrefix+"ssd"]
```

Comment:
Please use `GetCapacityKeyPrefix()`.
```suggestion
			val, ok := node.Annotations[topolvm.GetCapacityKeyPrefix()+"ssd"]
```